static/js/containers/pages/register/RegisterDeniedPage_test.js

```diff
@@ -42,17 +43,25 @@ describe("RegisterDeniedPage", () => {
 
   //
   ;[true, false].forEach(hasError => {
-    it(`${hasError ? "does" : "does not"} show an error message if ${
-      hasError ? "present in" : "absent from"
-    } the query string`, async () => {
-      const { inner } = await renderPage()
+    it(`${shouldIf(hasError)} show an error message if ${isIf(
+      hasError
+    )} the query string`, async () => {
```

Comment:
Nit of all nits: should be `in the query string` rather than just `the query string`